pynamodb/models.py

```diff
@@ -232,12 +237,16 @@ class Model(AttributeContainer):
     _index_classes = None
     DoesNotExist = DoesNotExist
 
-    def __init__(self, hash_key=None, range_key=None, **attributes):
+    def __init__(self, hash_key=None, range_key=None, previously_saved=False, **attributes):
         """
         :param hash_key: Required. The hash key for this object.
         :param range_key: Only required if the table has a range key attribute.
         :param attrs: A dictionary of attributes to set on this object.
         """
+        # This is for the default_for_new attribute value.  We assume an attribute
+        # is "new" if it has not been previously saved.  previously_saved will be set to true
+        # when deserializing JSON that dynamoDB returns
+        self._previously_saved = previously_saved
```

Comment:
Where is this used? Is this different from `is_new_object`?

Reply:
you're right.. I only use it in the constructor.  This is a vestige from an earlier implementation